src/vendor/rand.cc

```diff
@@ -106,11 +106,6 @@ constexpr void SEED(const T x0, const T x1, const T x2) {
   SET3(x, x0, x1, x2), SET3(a, A0, A1, A2), c = C;
 }
 
-template <typename T>
-constexpr T HiBit(const T x) {
```

Comment:
This function appears to be unused.

Reply:
this is vendored ( pasted from other project )

Maybe we can disable lint for "vendor" dir?

Reply:
Even though it's in the vendored directory, I believe it's more of a modern C++ re-implementation based on the comment below. Therefore, removing the unused function should be acceptable, at least NFC.

https://github.com/apache/kvrocks/blob/5ecf023bcf0a92e23982fef1710a53f495cfef9a/src/vendor/rand.cc#L44-L47